rest/komand_rest/actions/delete/action.py

```diff
@@ -19,12 +19,12 @@ def run(self, params={}):
             method="DELETE",
             path=params.get(Input.ROUTE),
             data=params.get(Input.BODY, {}),
-            headers=params.get(Input.HEADERS, {})
+            headers=params.get(Input.HEADERS, {}),
         )
 
         return {
             Output.BODY_OBJECT: Common.body_object(response),
             Output.BODY_STRING: response.text,
```

Comment:
In case of error... Will the response always have text?

Reply:
Hey @jmcadams-r7,
I've never encountered a case where response.text failed to return something - it might return an empty string, but it shouldn't cause an Exception 🙂

Reply:
Thanks